cmd/spot/main.go

```diff
@@ -163,12 +163,14 @@ func run(opts options) error {
 }
 
 // runTasks runs all tasks in playbook by default or a single task if specified in command line
-func runTasks(ctx context.Context, taskName string, targets []string, r *runner.Process) error {
-	// run a single task if specified
-	if taskName != "" {
-		for _, targetName := range targetsForTask(targets, taskName, r.Playbook) {
-			if err := runTaskForTarget(ctx, r, taskName, targetName); err != nil {
-				return err
+func runTasks(ctx context.Context, taskNames, targets []string, r *runner.Process) error {
+	// run specified tasks if there is any
+	if len(taskNames) > 0 {
```

Comment:
i think this condition can be removed as the for below won't do anything on the empy list anyway

Reply:
There is a return inside this if statement which stops function execution if tasks are specified and before running all tasks in the playbook. So I think this condition is needed here. Or I misunderstood something?